contrib/autostyle.lua

```diff
@@ -87,15 +87,17 @@ function autostyle_apply_one_image (image)
 
   -- Apply the style to image, if it is tagged
   local ok,auto_dr_attr= pcall(exiftool_attribute,image.path .. '/' .. image.filename,tag)
+  --darktable.print_error("dr_attr:" .. auto_dr_attr)
   -- If the lookup fails, stop here
   if (not ok) then
+    darktable.print("Couldn't get attribute" )
     return
   end
   if auto_dr_attr==value then
---	  darktable.print("Image " .. image.filename .. ": autostyle automatically applied " .. darktable.preferences.read("autostyle","exif_tag","string") )
+	  darktable.print_error("Image " .. image.filename .. ": autostyle automatically applied " .. darktable.preferences.read("autostyle","exif_tag","string") )
```

Comment:
Changing darktable.print to darktable.print_error takes feedback away from the user and puts it in the debugging stream, which the user isn't likely to look at unless there is a problem and they are know to look there.